src/tests/conformance/query/get_by_hash.rs

```diff
@@ -85,10 +85,10 @@ async fn c008_TM_HAVE_TRANSACTIONS_query_for_transactions_after_have_transaction
     // ZG-CONFORMANCE-008
     // Create stateful node.
     let target = TempDir::new().expect("unable to create TempDir");
-    let mut node = build_stateful_builder(target.path().to_path_buf())
+    let mut node = build_stateful_builder()
         .expect("unable to get stateful builder")
         // TODO consider moving `log_to_stdout` to builder state.
-        .start(false)
+        .start(target.path().to_path_buf(), NodeType::Stateful, false)
         .await
```

Comment:
A bit of code smell here: function has 'stateful' in its name and then we also give `NodeType::Stateful`.

Reply:
I agree. I wouldn't say I like the name of this function above. It just does some extra work to prepare some stuff for a stateful node. We could delete it and do those extra things in every test.

```
pub fn build_stateful_builder() -> anyhow::Result<NodeBuilder> {
    Ok(Node::builder()
        .network_id(TESTNET_NETWORK_ID)
        .validator_token(get_validator_token(0))
        .add_args(vec![
            "--valid".into(),
            "--quorum".into(),
```